gprofiler/profilers/java.py

```diff
@@ -279,6 +280,72 @@ def get_ap_version() -> str:
     return Path(resource_path("java/async-profiler-version")).read_text()
 
 
+@functools.lru_cache(maxsize=1024)
+def get_jvm_flags(
+    process: Process,
+) -> List[Tuple[Optional[str], Optional[str], Optional[str], str, Optional[List[str]]]]:
```

Comment:
I'm not sure about the types here: Why are the flag name, type, and value optional? Even the kind should always have some value afaict from the code.

Reply:
All of them are regex match groups they could be None, Ill change it to also handle this comment https://github.com/Granulate/gprofiler/pull/660#discussion_r1084952682 and Ill add to the list only flags that were parsed correctly